.cicd/platforms/amazon_linux-2-unpinned.dockerfile

```diff
@@ -5,7 +5,7 @@ RUN yum update -y && \
     yum install -y which git sudo procps-ng util-linux autoconf automake \
     libtool make bzip2 bzip2-devel openssl-devel gmp-devel libstdc++ libcurl-devel \
     libusbx-devel python3 python3-devel python-devel libedit-devel doxygen \
-    graphviz clang patch llvm-devel llvm-static
+    graphviz clang patch llvm-devel llvm-static vim-common jq
```

Comment:
I chose to include `xxd` here and `jq` on the other pull request so that these tools would be available in the container when the container is pulled or SSH'ed into for debugging. heifner and jerstern were doing this today, and I think this practice will become more common.